tests/integration/docusaurus/deployment_patterns/aws_cloud_storage_pandas.py

```diff
@@ -269,10 +269,6 @@
 )
 # </snippet>
 
-# <snippet name="tests/integration/docusaurus/deployment_patterns/aws_cloud_storage_pandas.py run checkpoint">
-checkpoint_result = checkpoint.run()
-# </snippet>
-
 assert not checkpoint_result.success
```

Comment:
Same thing here - let's only delete the surrounding tags and not the encapsulated code